thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/template.hbs

```diff
@@ -228,17 +230,19 @@
                   <li class="te-anomaly-table__list--lighter">{{anomaly.durationStr}}</li>
                 </ul>
                </td>
-               <td class="te-anomaly-table__cell">
-                <ul class="te-anomaly-table__list">
-                 {{#each anomaly.dimensionList as |dimension|}}
-                    <li class="te-anomaly-table__list--smaller">
-                      {{dimension.dimensionKey}}: <span class="stronger">{{dimension.dimensionVal}}</span>
-                    </li>
-                 {{else}}
-                    -
-                 {{/each}}
-                </ul>
-               </td>
+               {{#if alertHasDimensions}}
+                 <td class="te-anomaly-table__cell">
+                  <ul class="te-anomaly-table__list">
+                   {{#each anomaly.dimensionList as |dimension|}}
+                      <li class="te-anomaly-table__list--smaller">
```

Comment:
looks like you have a BEM modifier on a child element. 
The correct usage would be creating a child class `list-item` (modifier might not be needed), or change the class name so that it applies to the list item 
`te-anomaly-table__list-item te-anomaly-table__list-item--smaller`

Reply:
Thanks, I will modify.